pull-request/src/person.js

```diff
@@ -3,7 +3,7 @@ function Person(first, last) {
     this.lastName = last;
     this.age = null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use null instead. ([javascript:S2138](http://localhost:9750/coding_rules#rule_key=javascript%3AS2138))

[See it in SonarQube](http://localhost:9750/project/issues?id=training%3Apull-request&pullRequest=51&issues=AWjNl6jM-op2ExCg4NRB&open=AWjNl6jM-op2ExCg4NRB)

[ISSUE_KEY]: AWjNl6jM-op2ExCg4NRB